JeMPI_Apps/JeMPI_UI/src/services/ApiClient.ts

```diff
@@ -214,8 +214,15 @@ export class ApiClient {
   ): Promise<ApiSearchResult<AnyRecord>> {
     const isCustomSearch = '$or' in request
     const endpoint = `${
-      isCustomSearch ? ROUTES.POST_CUSTOM_SEARCH : ROUTES.POST_SIMPLE_SEARCH
-    }/${isGoldenOnly ? 'golden' : 'patient'}`
+      isGoldenOnly
+        ? isCustomSearch
+          ? ROUTES.POST_CUSTOM_GOLDEN_SEARCH
+          : ROUTES.POST_SIMPLE_GOLDEN_SEARCH
+        : isCustomSearch
+        ? ROUTES.POST_SIMPLE_INTERACTION_PATIENT_SEARCH
+        : ROUTES.POST_CUSTOM_INTERACTION_PATIENT_SEARCH
+    }`
+
```

Comment:
@harmochiky please review the use of the interaction patient search. To my knowledge, the ui does not use this anymore. If so, please remove this and the need for checking on isGoldenOnly property

Reply:
Resolved @MatthewErispe